### PR TITLE
Cleanup unused parameter

### PR DESCRIPTION
**Pull request by MorrisJobke**
* Remove crypto and timefactory from UsersController
* Remove config from AutoCompleteController
* Remove fromMailAddress from MailSettingsController
   * Was removed in https://github.com/nextcloud/server/pull/4379 (0a54d5a) and https://github.com/nextcloud/server/pull/4380 (bae64e8)

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

File: lib/private/AppFramework/DependencyInjection/DIContainer.php [hunk 1]
```diff
@@ -152,10 +152,6 @@ public function __construct($appName, $urlParams = array(), ServerContainer $ser
 			return $c->query('ServerContainer')->getWebRoot();
 		});
 
-		$this->registerService('fromMailAddress', function() {
```
Comment (nickvergessen) [inline lib/private/AppFramework/DependencyInjection/DIContainer.php]: this might break apps

Comment (MorrisJobke) [reply lib/private/AppFramework/DependencyInjection/DIContainer.php]: It is only used by us in this one occurence ... not sure we want to provide this. (searched across all apps in the appstore)

Comment (MorrisJobke) [reply lib/private/AppFramework/DependencyInjection/DIContainer.php]: But we could also keep this ofc

Comment (rullzer) [reply lib/private/AppFramework/DependencyInjection/DIContainer.php]: Kill it for now then. Just be sure the add it to the changes for devs

Comment (MorrisJobke) [reply lib/private/AppFramework/DependencyInjection/DIContainer.php]: Done in #7827

**Comment by MorrisJobke** [comment]
@nickvergessen @rullzer Can we get this in?

**Review by rullzer — APPROVED** [review]

**Pull request merged**